### ISSUE TITLE
Official version support

### ISSUE THREAD

**Issue by phobos2077** [RadMuravTeam/Fallout-Sonora-ENG#116]
The readme mentions that only "Extended" version is supported. Is there any plans to make it compatible with the official version?

Would like to have vanilla experience for the first playthrough.

**Comment by roginvs** [comment]
Yep, it will be good to have translation from the latest vanilla version 1.13 (+ DLC 1.13)

**Comment by FicMon** [comment]
Not sure about official version support since the data structuture of foxxs rendition of sonora is different from the official vanilla release.

**Comment by phobos2077** [comment]
> Not sure about official version support since the data structuture of foxxs rendition of sonora is different from the official vanilla release.

What data structure?

**Comment by FicMon** [comment]
By data structure I mean the way files in the data folder are organized. From what I was told from other team members foxxs repack of sonora organizes files differently from the vanilla version and I did some testing months ago with trying to use the translation eith vanilla sonora but I got weird errors.

**Comment by phobos2077** [comment]
> By data structure I mean the way files in the data folder are organized. From what I was told from other team members foxxs repack of sonora organizes files differently from the vanilla version and I did some testing months ago with trying to use the translation eith vanilla sonora but I got weird errors.

It's not important. The game uses virtual file system and all translation files are in the same place regardless. All you need to do is provide translated msg and frm files, then it's the matter of how you make the release package. Your git repo will remain the same regardless.
